packages/ifpack2/src/CMakeLists.txt

```diff
@@ -236,6 +236,14 @@ IF(Ifpack2_ENABLE_EXPLICIT_INSTANTIATION)
     APPEND_GLOBAL_SET(IFPACK2_ETI_CLASSES Krylov)
   ENDIF()
 
+  IF(Ifpack2_ENABLE_ShyLUFastILU)
```

Comment:
@brian-kelley It would be confusing to have some classes left out of ETI.  Also, Ifpack2::Factory necessarily requires instantiating all solvers that they can build, so you might as well plug the new classes into ETI.

The other advantage of enabling ETI for these classes, is that you could consider use of the "pImpl" idiom to avoid including ShyLU headers in Ifpack2 _decl.hpp headers.